e2e/cypress/integration/api/iam/05_iam_integration.spec.ts

```diff
@@ -0,0 +1,205 @@
+import { describeIfIAMV2p1 } from '../../constants';
+
+describeIfIAMV2p1('assigning projects', () => {
+  let adminIdToken = '';
+  let apiToken = '';
+
+  const modifyError = 'cannot modify a project for this object';
+  const cypressPrefix = 'iam-integration';
+  const resource = {
+    id: `${cypressPrefix}-${Cypress.moment().format('MMDDYYhhmm')}`,
+    name: 'test',
+    projects: []
+  };
+  const notFoundProjectId = 'not-found-project';
+  const authorizedProject1 = `${cypressPrefix}-authorized-project-1`;
+  const authorizedProject2 = `${cypressPrefix}-authorized-project-2`;
+  const unauthorizedProjectId = `${cypressPrefix}-unauthorized-project`;
+  const policyId = `${cypressPrefix}-policy`;
+  const tokenId = `${cypressPrefix}-token`;
+  const iamResourcesToTest = ['roles', 'tokens', 'teams'];
+  const objectsToCleanUp = iamResourcesToTest.concat(['projects']);
+
+  before(() => {
+    cy.adminLogin('/').then(() => {
+      adminIdToken = JSON.parse(<string>localStorage.getItem('chef-automate-user')).id_token;
+
+      // TODO cleanup everything in resources + projects
+      cy.cleanupV2IAMObjectsByIDPrefixes(adminIdToken, cypressPrefix, objectsToCleanUp);
+
+      for (const id of [unauthorizedProjectId, authorizedProject1, authorizedProject2]) {
+        cy.request({
+          auth: { bearer: adminIdToken },
+          method: 'POST',
+          url: '/apis/iam/v2beta/projects',
+          failOnStatusCode: false,
+          body: {
+            id: id,
+            name: id
+          }
+        }).then((resp) => {
+          expect([200, 409]).to.include(resp.status);
```

Comment:
This has been in the back of my mind for some time, so I finally looked up the right function.
We have been using `expect(x)` incorrectly. `expect(x)` is supposed to mean `expect(<actual-result>)` not `expect(<expected-result>)`. Let's fix this:
```suggestion
          expect(resp.status).to.be.oneOf([200, 409]);
```

Reply:
ah didn't know `oneOf` 👍